near-plugins-derive/tests/upgradable.rs

```diff
@@ -203,19 +203,17 @@ impl Setup {
     }
 }
 
-/// Panics if the conversion fails.
-fn convert_code_to_crypto_hash(code: &[u8]) -> CryptoHash {
-    near_sdk::env::sha256(code)
-        .try_into()
-        .expect("Code should be converted to CryptoHash")
+/// Converts code to a base58-encoded CryptoHash string.
+fn convert_code_to_crypto_hash(code: &[u8]) -> String {
```

Comment:
This is not crypto_hash; right now, we are returning CryptoHash. It's not very important, but maybe we could rename it to convert_code_to_bs58_encoded_hash or something like that?

Reply:
cc @r-near